app/controllers/concerns/api/current/sync_encounter_observation.rb

```diff
@@ -0,0 +1,60 @@
+module Api::Current::SyncEncounterObservation
+  extend ActiveSupport::Concern
+  included do
+
+    def merge_encounter_observation(key, params)
+      ActiveRecord::Base.transaction do
+        if FeatureToggle.enabled?('SYNC_ENCOUNTERS')
+          # this will always return a single blood_sugar
```

Comment:
Since this is a concern, this comment can probably mention that it will return a single observation rather than a single `blood_sugar`?